score-lang-cli/src/main/java/org/openscore/lang/cli/SlangBanner.java

```diff
@@ -38,13 +42,13 @@ public String getBanner() {
 			sb.append("Slang");
 		}
 		sb.append(System.lineSeparator());
-		sb.append(SlangCLI.getVersion());
+		sb.append(getVersion());
 		return sb.toString();
 	}
 
 	@Override
 	public String getVersion() {
-		return SlangCLI.getVersion();
+		return slangCLI.getSlangVersion();
```

Comment:
why ?? why not add instead :
@Value("${slang.version}")
private String slangVersion;